src/main.cpp

```diff
@@ -1662,6 +1662,16 @@ void reset_creature_max_levels(void)
     }
 }
 
+void reset_hand_rules(void)
+{
+    struct DungeonAdd* dungeonadd;
+    for (int i = 0; i < DUNGEONS_COUNT; i++)
+    {
+        dungeonadd = get_dungeonadd(i);
+        memset(dungeonadd->hand_rules, 0, sizeof(dungeonadd->hand_rules));
+    }
+}
+
```

Comment:
main is currently still full of stuff than really should be moved around to other places, I'd probably put this in power_hand.c

Reply:
or maybe even make a power_hand rules file because power hand is getting on the long side